### PR TITLE
Correct an obvious typo

### LINKED ISSUE #13566: typo in Syntax Highlighting section
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll#syntax-highlighting

### What changes are you suggesting?

It states that if you set the settings to "Pygments", the website will use "Rouge" instead of "Rouge". Obviously, it should be "Pygments" instead of "Rouge"

### Additional information

I'm creating the issue only because the PR guide state it's best to do so. In this case, I don't believe it actually adds values

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
### Why:

Closes #13566
### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ For a list of supported plugins, see "[Dependency versions](https://pages.github
...
+By default, code blocks on your site will be highlighted by Jekyll. Jekyll uses the [Rouge](https://github.com/jneen/rouge) highlighter, which is compatible with [Pygments](http://pygments.org/). If you specify Pygments in your *_config.yml* file, Pygments will be used instead. Jekyll cannot use any other syntax highlighter, and you'll get a page build warning if you specify another syntax highlighter in your *_config.yml* file. For more information, see "[About Jekyll build errors for {% data variables.product.prodname_pages %} sites](/articles/about-jekyll-build-errors-for-github-pages-sites)."
```
Comment (AnilSeervi) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll.md:115]:
```suggestion
By default, code blocks on your site will be highlighted by Jekyll. Jekyll uses the [Rouge](https://github.com/jneen/rouge) highlighter, which is compatible with [Pygments](http://pygments.org/). Pygments has been deprecated and not supported in Jekyll 4 and if you specify Pygments in your *_config.yml* file, Rouge will be used as the fallback instead. Jekyll cannot use any other syntax highlighter, and you'll get a page build warning if you specify another syntax highlighter in your *_config.yml* file. For more information, see "[About Jekyll build errors for {% data variables.product.prodname_pages %} sites](/articles/about-jekyll-build-errors-for-github-pages-sites)."
```

Comment (AnilSeervi) [reply content/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll.md:115]: > Using Pygments has been deprecated and is not supported in Jekyll 4; the configuration setting highlighter: pygments now automatically falls back to using Rouge which is written in Ruby and 100% compatible with stylesheets for Pygments.

Refer - https://jekyllrb.com/docs/liquid/tags/#code-snippet-highlighting

**Review by AnilSeervi — COMMENTED** [review]
Jekyll won't use Pygments

**Comment by Arthur-Milchior** [comment]
Thanks for the rephrasing and explanation.
The way it was written, I understood that if you indicated "pygments" it will use "rouge" instead of "rouge". Now I understand it means that "it will still uses Rouge instead of the required Pygments". I like your new sentence as it has no more ambiguity

**Comment by ramyaparimi** [comment]
@Arthur-Milchior 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Bigg29 — COMMENTED** [review]
Thanks!

File: content/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ For a list of supported plugins, see "[Dependency versions](https://pages.github
...
+By default, code blocks on your site will be highlighted by Jekyll. Jekyll uses the [Rouge](https://github.com/jneen/rouge) highlighter, which is compatible with [Pygments](http://pygments.org/). Pygments has been deprecated and not supported in Jekyll 4 and if you specify Pygments in your *_config.yml* file, Rouge will be used as the fallback instead. Jekyll cannot use any other syntax highlighter, and you'll get a page build warning if you specify another syntax highlighter in your *_config.yml* file. For more information, see "[About Jekyll build errors for {% data variables.product.prodname_pages %} sites](/articles/about-jekyll-build-errors-for-github-pages-sites)."
```
Comment (skedwards88) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll.md:115]:
```suggestion
By default, code blocks on your site will be highlighted by Jekyll. Jekyll uses the [Rouge](https://github.com/jneen/rouge) highlighter, which is compatible with [Pygments](http://pygments.org/). Pygments has been deprecated and not supported in Jekyll 4. If you specify Pygments in your *_config.yml* file, Rouge will be used as the fallback instead. Jekyll cannot use any other syntax highlighter, and you'll get a page build warning if you specify another syntax highlighter in your *_config.yml* file. For more information, see "[About Jekyll build errors for {% data variables.product.prodname_pages %} sites](/articles/about-jekyll-build-errors-for-github-pages-sites)."
```

**Review by skedwards88 — DISMISSED** [review]
Thanks for this PR! I made a suggestion to split the addition into two sentences. I'll go ahead an incorporate that change, then get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**